static/js/common/upload-addon.js

```diff
@@ -27,7 +27,7 @@
     }
 
     $.fn.addonUploader = function( options ) {
-        var settings = {'filetypes': ['xpi', 'jar', 'xml'], 'getErrors': getErrors, 'cancel': $()};
+        var settings = {'filetypes': ['zip', 'xpi', 'jar', 'xml'], 'getErrors': getErrors, 'cancel': $()};
```

Comment:
We should remove `.json` then.

On Tue, May 31, 2016 at 4:12 AM, Andrew Williamson <notifications@github.com
> wrote:

> In static/js/common/upload-addon.js
> <https://github.com/mozilla/addons-server/pull/2785#discussion_r65163100>:
>
> > @@ -27,7 +27,7 @@
> >      }
> >
> >      $.fn.addonUploader = function( options ) {
> > -        var settings = {'filetypes': ['xpi', 'jar', 'xml'], 'getErrors': getErrors, 'cancel': $()};
> > +        var settings = {'filetypes': ['zip', 'xpi', 'jar', 'xml'], 'getErrors': getErrors, 'cancel': $()};
>
> .json was added for webapps
> https://bugzilla.mozilla.org/show_bug.cgi?id=676887
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mozilla/addons-server/pull/2785/files/de8551cf023af50715ca09888e937f8203bf6fe3#r65163100>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALT8wUdWfsL3X9yYelpezRbYSBEuyxQUks5qHBeYgaJpZM4Ioo0J>
> .
>